src/ModelInit.jl

```diff
@@ -464,7 +464,7 @@ function limit_to_object_data(
   # Get the sources that overlap with this object.
   relevant_sources = Int[]
   for b = 1:length(blob), tile_sources in mp.tile_sources[b]
-    if s_original in tile_sources > 0
```

Comment:
We probably didn't mean to compare a boolean to an int before, but for whatever reason 0.4 doesn't complain, while 0.5 stops.